wolox-books/src/app/register/register.component.html

```diff
@@ -0,0 +1,34 @@
+<form [formGroup]="rForm" (ngSubmit)="addPost(rForm.value)" class="form">
+  <div class="form-container">
+    <header class="header">
+      <img src="assets/wolox_logo.svg" class="header-logo" alt="Wolox logo">
+      <h1 class="header-title">BOOKS</h1>
+    </header>
+    <div class="form-signup">
+      <label class="form-field">First name
+        <input type="text" formControlName="firstName" class="form-input">
+      </label>
```

Comment:
The `label` tag is just for the label that goes with the `input` field. In this case, just use this tag for the `First name` text